src/main/webapp/WEB-INF/web.xml

```diff
@@ -5,9 +5,9 @@
          version="3.1">
 
     <display-name>inquirer</display-name>
-
-
-
+    <listener>
+        <listener-class>org.apache.tiles.web.startup.simple.SimpleTilesListener</listener-class>
+    </listener>
```

Comment:
<url-pattern>/</url-pattern> listen ALL requests to application. Use <url-pattern>/hello</url-pattern> e.g. and access to page will be http://localhost:8080/inquirer/hello